addon/index.js

```diff
@@ -164,6 +166,17 @@ Object.defineProperty(DS, 'normalizeModelName', {
   value: normalizeModelName
 });
 
-Ember.lookup.DS = DS;
+Object.defineProperty(global, 'DS', {
```

Comment:
ping @rwjblue. Do you have time to work on this? If not I can try to find someone else to get this pr over the finish line.